### DISCUSSION TITLE
[A-L] (2023/24) Foglio 6 - Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#145] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-11-09 172721](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/fcd0420e-171f-4393-82d9-3fd22d02928e)

**Comment by CasuFrost** [comment]
Se un gruppo $G$ è di ordine $p$, allora è isomorfo a $\mathbb{Z}_p$, essendo $\mathbb{Z}_p$ ciclico e generato da 1, allora anche $G$ è ciclico.

**Reply by interestingmaneuver** [reply]
Quest'affermazione non è corretta, in tal caso si avrebbe che ogni gruppo finito è ciclico (perché $\mathbb{Z}_n$ lo è per ogni $n$), ma ad esempio $(\mathbb{Z}_8^*, \cdot) \not\cong \mathbb{Z}_4$. 

Si ha che se un gruppo $G$ **è ciclico** e ha ordine $n < \infty$, allora è isomorfo a $\mathbb{Z}_n$.

**Reply by CasuFrost** [reply]
grazie

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/5bd339e4-0111-4777-92a6-476cde51558e)

**Comment by cornflexxx** [comment]
### Soluzione dal libro _Algebra di  I.N.  Herstein_

Per prima cosa, $G$ non ha sottogruppi $H$ non banali: $o(H)$ deve essere infatti un divisore di $o(G)=p$, per cui le possibilità sono $o(H)=1$ oppure $o(H)=p$. Nel primo caso , $H=\left\langle  e\right\rangle$ nel secondo $H=G$. Sia ora $a$ un elemento di $G$ con $a\neq e$, e sia $H=\left\langle  a\right\rangle$. $H$ è un sottogruppo di $G$ e $H\neq \left\langle e \right\rangle$ in quanto $a\in H$  e $a\neq e$.  Allora $H=G$, e dunque $G$ è ciclico e ogni suo elemento è una potenza di $a$.

(Con $e$ si intende l'elemento neutro)
